.idea/.generators

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Należy to zrobić następująco:
`git rm --cached -r .idea`
Następnie dodać wpis `.idea` do pliku `.gitignore`